cmd/registry-replacer/main.go

```diff
@@ -600,6 +635,88 @@ func pruneReplacements(config *api.ReleaseBuildConfiguration, filter asDirective
 	return utilerrors.NewAggregate(errs)
 }
 
+// when pruning unused base images we'll check the fully-resolved config to make sure an image is not used directly in
```

Comment:
Nit: It is convention to start comments on named things with the name of the thing, ref https://golang.org/doc/effective_go#commentary

> If every doc comment begins with the name of the item it describes, you can use the doc subcommand of the go 
> tool and run the output through grep

Reply:
Ah, I thought that only applied to exported functions.